api/api.go

```diff
@@ -62,4 +62,5 @@ func Serve(router *mux.Router, store *storage.Storage, pool *worker.Pool) {
 	sr.HandleFunc("/entries", handler.setEntryStatus).Methods(http.MethodPut)
 	sr.HandleFunc("/entries/{entryID}", handler.getEntry).Methods(http.MethodGet)
 	sr.HandleFunc("/entries/{entryID}/bookmark", handler.toggleBookmark).Methods(http.MethodPut)
+	sr.HandleFunc("/entries/{entryID}/download", handler.fetchContent).Methods(http.MethodGet)
```

Comment:
It would be probably better to name the endpoint `/fetch-content` instead of `/download`. It's more specific and avoid possible confusions.

Reply:
Sure